controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
@aslakknutsen I also had doubts about the same. My concern was if I get tx.NotFoundError, It is an ERROR, and it should not be ignored and proceeded.

```
tx = r.db.Where("execution_order > ?", order).Order("execution_order", true).Last(&Item)

If tx.RecordNotFound() {
                  return Item.ID, nil, tx.Error
}
```

@aslakknutsen @kwk does this look like a proper way to handle the case?